src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -471,7 +472,11 @@ export class ResultsViewPageStore {
 
     readonly putativeDriverFilteredCaseAggregatedDataByUnflattenedOQLLine = remoteData<{
         cases: CaseAggregatedData<AnnotatedExtendedAlteration>,
-        oql: UnflattenedOQLLineFilterOutput<AnnotatedExtendedAlteration>
+        oql: UnflattenedOQLLineFilterOutput<AnnotatedExtendedAlteration>,
+        list?: {
+            cases: CaseAggregatedData<AnnotatedExtendedAlteration>,
+            oql: OQLLineFilterOutput<object>
+        }[]
```

Comment:
could you pull this out into a type def and add a comment explaining it?

Reply:
Right, this shape of object occurs all over the page store; it would be useful to give it a name.

Reply:
Unless you're referring to the meaning of the `list:` property itself, which I now realise not to be exactly self-explanatory either if you haven't just written the code that interprets the OQL parser tree. I'd have time to clarify that on Monday.

Reply:
I extracted the interfaces and added comments to them. In doing so I found that I'd accidentally left in an unused function before I made the PR, and I removed that function.